### PR TITLE
3419 suspicious case to cases

### PR DESCRIPTION
**Pull request by cazacmarin**

### CODE REVIEW

File: sormas-api/src/main/resources/captions.properties [hunk 1]
```diff
@@ -2152,3 +2157,17 @@ VaccinationInfo.vaccineInn=INN
 VaccinationInfo.vaccineBatchNumber=Batch number
 VaccinationInfo.vaccineUniiCode=UNII code
 VaccinationInfo.vaccineAtcCode=ATC code
+
+VaccinationInfo.vaccination=Vaccination status for this disease
```
Comment (leventegal-she) [inline sormas-api/src/main/resources/captions.properties:2161]: these are not needed here

File: sormas-api/src/main/resources/enum.properties [hunk 2]
```diff
@@ -1369,6 +1369,26 @@ Vaccine.SANOFI_GSK=Sanofi-GSK
 Vaccine.UNKNOWN=Unknown
 Vaccine.OTHER=Other
 
+# VaccineManufacturer
```
Comment (leventegal-she) [inline sormas-api/src/main/resources/enum.properties:1372]: please delete these ones and from all other enum.properties files

File: sormas-api/src/main/resources/validations_de-CH.properties [hunk 3]
```diff
@@ -140,4 +140,5 @@ externalJournalPersonValidationPhoneTaken = Die Telefonnummer wird bereits für
 externalJournalPersonValidationEmailTaken = Die E-Mail-Adresse wird bereits für das Symptomtagebuch verwendet.
 externalJournalPersonValidationNoEmailOrPhone = Bitte geben Sie eine E-Mail-Adresse oder eine Telefonnummer an.
 campaignFormMetaValidationUnexpectedError = Beim Überprüfen dieses Formulars ist ein unerwarteter Fehler aufgetreten. Bitte leiten Sie die folgende Fehlermeldung an einen Systemadministrator weiter\: %s
+vaccineDosesFormat=Impfstoffdosis sollte eine Zahl zwischen 1 und 10 sein
```
Comment (leventegal-she) [inline sormas-api/src/main/resources/validations_de-CH.properties:143]: please delete this and from all other validations.properties files

File: sormas-app/app/src/main/res/layout/dialog_exposure_read_layout.xml [hunk 4]
```diff
@@ -163,6 +163,11 @@
 
                 </LinearLayout>
 
+                <de.symeda.sormas.app.component.controls.ControlTextReadField
```
Comment (leventegal-she) [inline sormas-app/app/src/main/res/layout/dialog_exposure_read_layout.xml:166]: this has nothing to do with your task, please do a proper merge with the development branch

File: sormas-app/app/src/main/res/layout/dialog_exposure_read_layout.xml [hunk 5]
```diff
@@ -441,6 +446,11 @@
...
```
Comment (leventegal-she) [inline sormas-app/app/src/main/res/layout/dialog_exposure_read_layout.xml:449]: same as above

File: sormas-app/app/src/main/res/layout/fragment_case_edit_layout.xml [hunk 6]
```diff
@@ -785,6 +821,79 @@
                         style="@style/ControlSecondOfTwoColumnsStyle" />
                 </LinearLayout>
 
+                <LinearLayout
```
Comment (leventegal-she) [inline sormas-app/app/src/main/res/layout/fragment_case_edit_layout.xml:824]: This has nothing to do with your ticket, please do a proper merge with the development branch

File: sormas-base/setup/sormas.properties [hunk 7]
```diff
@@ -199,3 +199,6 @@ sms.auth.secret=
 
 # Control the creation of the default users
 # createDefaultUsers=true
+
```
Comment (leventegal-she) [inline sormas-base/setup/sormas.properties:202]: please delete this, or do a proper merge with the development branch

**Review by leventegal-she — CHANGES_REQUESTED** [review]
Didn't see the logic for "If the answer is “suspicion not confirmed” then change case classification to “no case”."

**Comment by MateStrysewske** [comment]
@cazacmarin @leventegal-she I guess this pull request can be closed because you opened a new one to address the issues, right?

**Pull request closed without merging**